src/index.ts

```diff
@@ -302,13 +303,23 @@ export function apply(ctx: Context, config: Config) {
         }
       })()
 
+      const getResponseType = () => {
+        if (config.type === "stable-horde" || config.type === "sd-webui") {
+          return "json"
+        }
+        return "arraybuffer"
+      }
+
       const getPayload = () => {
         switch (config.type) {
           case 'login':
           case 'token':
           case 'naifu': {
             parameters.sampler = sampler.sd2nai(options.sampler)
             parameters.image = image?.base64 // NovelAI / NAIFU accepts bare base64 encoded image
+            // The latest interface changes uc to negative_prompt, so that needs to be changed here as well
+            parameters.negative_prompt = parameters.uc
```

Comment:
I thought you'd better change all the `.uc` into `.negative_prompt` here.
And another concern is, the `naifu` method still following the official NovelAI here or remains as before?

Reply:
Sorry, I thought naifu was part of novelai's method, I will modify it